### PR TITLE
Displays the logs from the logger

### PR DESCRIPTION
**Pull request by NekoSelena**
Needs solution for the `Logger.hpp` include

### CODE REVIEW

File: cubic-server/interface/LogsInterface.cpp [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #include <string>
 #include <iostream>
 #include <fstream>
+#include <queue>
 #include <unistd.h>
 
-//#include "Logger.hpp"
+#include "../logging/Logger.hpp"
```
Comment (NekoSelena) [inline cubic-server/interface/LogsInterface.cpp:7]: Please leave a comment here if you know how to fix this include

**Review by Trompettesib — DISMISSED** [review]

File: cubic-server/interface/LogsInterface.cpp (hunk 1 above)
Comment (huntears) [reply cubic-server/interface/LogsInterface.cpp:7]: Shouldn't `#include "logging/Logger.hpp"` work alright ?

Comment (NekoSelena) [reply cubic-server/interface/LogsInterface.cpp:7]: I'll try and see if it works

**Review by Trompettesib — APPROVED** [review]
👌

**Review by huntears — APPROVED** [review]

**Review by Trompettesib — APPROVED** [review]
👌

**Pull request merged**